mono/mini/method-to-ir.c

```diff
@@ -6897,7 +7017,25 @@ mono_method_to_ir (MonoCompile *cfg, MonoMethod *method, MonoBasicBlock *start_b
 		if (cfg->verbose_level > 3)
 			printf ("converting (in B%d: stack: %d) %s", cfg->cbb->block_num, (int)(sp - stack_start), mono_disasm_code_one (NULL, method, ip, NULL));
 
-		switch (*ip) {
+		MonoOpcodeParameter parameter;
+		const MonoOpcodeInfo* info = mono_opcode_decode (ip, op_size, il_op, &parameter);
```

Comment:
Could you benchmark if compilation times significantly change because of this?
Something as simple as compiling the whole of corlib (--compile-all) should do it.
This was a perf issue I hit in the past.

Reply:
I doubt it but yes I will.
There is an unfortunate double  array lookup that could be brought back to one (was zero).

Reply:
Timgings looked identical/noisy/sometimes-one-faster-sometimes-the-other building mscorlib arm32 on builder.